src/cryptoadvance/specter/cli/cli_server.py

```diff
@@ -245,8 +233,8 @@ def run(debug=debug):
             app.run(debug=debug, **kwargs)
             stop_hidden_services(app)
         finally:
-            if app.controller is not None:
-                app.controller.close()
+            if app.specter.tor_controller is not None:
+                app.specter.tor_controller.close()
```

Comment:
```suggestion
            try:
                if app.specter.tor_controller is not None:
                    app.specter.tor_controller.close()
            except SpecterError as se:
                # no reason to break startup here
                pass    
```

Reply:
However, just a pass is probably not good enough. We should do a 
```
logger.error("Could not initialize tor-system")
```
Or something like that.